actionpack/lib/action_controller/renderer.rb

```diff
@@ -82,8 +82,9 @@ def initialize(controller, env, defaults)
     #   need to call <tt>.to_json</tt> on the object you want to render.
     # * <tt>:body</tt> - Renders provided text and sets content type of <tt>text/plain</tt>.
     #
-    # If no <tt>options</tt> hash is passed or if <tt>:update</tt> is specified, the default is
-    # to render a partial and use the second parameter as the locals hash.
+    # <tt>options</tt> hash is passed or if <tt>:update</tt> is specified, the default is to either call
```

Comment:
Looks like `If no` got lost. Think the wording is slightly clunky too. I think there's a version that we can write in half the length.